README.md

```diff
@@ -69,6 +69,10 @@ $ npm run test:cov
 
 Nest is an MIT-licensed open source project. It can grow thanks to the sponsors and support by the amazing backers. If you'd like to join them, please [read more here](https://docs.nestjs.com/support).
 
+[![Stargazers repo roster for @EddieHubCommunity/EddieBot](https://reporoster.com/stars/EddieHubCommunity/EddieBot)](https://github.com/EddieHubCommunity/EddieBot/stargazers)
```

Comment:
I'm not sure if we need to feature this, since these are displayed in there respective sections...
However, if we are adding these from reporoster, maybe the text above this, in the `Support` heading can be edited to reflect the current context that this is in regard to this project, and not Nest, moreover this isn't showcasing sponsors, rather people that forked/starred the project?